app-server/datamodel/src/main/scala/com/azavea/rf/datamodel/latest/schema/schema.scala

```diff
@@ -274,5 +311,5 @@ trait tables {
 }
 
 object Version{
-  def version = 10
+  def version = 11
```

Comment:
These all need to be bumped to 13 I think for me to be able to test this (essentially rebased from develop)